### ISSUE TITLE
wrote 0 records to TE.table.txt

### ISSUE THREAD

**Issue by CWYuan08** [adamewing/tldr#36]
Hi @adamewing @SarahBeecroft , we are hoping to run tldr with our ONT data to find TE insertions. 

I have run it with
tldr -b  alignment.sorted.bam  -e renamed.fa -r reference.dna.toplevel.fasta  --color_consensus --detail_output  -o TE
but I got an empty table at the end. 

From the error message, it looks like different numbers of clusters picked up from the pickles, but 0 records wrote to the table.

I am working with a non-model organism, and it is not simulated data.
Do you know how i can improve this? 

Thank you very much!

Best,
CW

**Comment by SignorLab** [comment]
I am having the exact same issue! Wish this thread hadn't been dead since March

**Comment by adamewing** [comment]
Apologies for not staying on top of issues. Could you let me know what kind of data you're using (ONT, PacBio), what genome the reads are aligned to (specific build would help) and what program was used to do the alignment (e.g. minimap2). Could you also run with `--debug` and pipe stdout+stderr to a file e.g.:
```
tldr (...options...) > tldr_debug.txt 2>&1
```

**Comment by SignorLab** [comment]
It is ONT data. I am aligning it to dsim 2.02 from flybase. The alignment was done with minimap2. I am currently running it with debug as you suggested.

**Comment by SignorLab** [comment]
The tldr debug file looks like this, this isn't every line but the whole output is just more lines like this:

/home/sasignor/anaconda3/envs/tldr_3/bin/tldr:4: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  __import__('pkg_resources').run_script('tldr==1.2.2', 'tldr')
2024-05-08 16:36:29,201 tldr started with command: /home/sasignor/anaconda3/envs/tldr_3/bin/tldr -b SZ129.alignment.sorted.bam -e ref/chakraborty_simulans_header.fasta -r /storehouse>
2024-05-08 16:36:29,201 output basename: SZ129.alignment.sorted
2024-05-08 16:36:29,214 base in ref/chakraborty_simulans_header.fasta, id DNAP:PROTOPB, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,214 base in ref/chakraborty_simulans_header.fasta, id DNAP:PROTOPB, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNACMCTransib:TransibN1DM, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNAhAT:hAT1NDP, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNAhAT:hAT1NDP, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,218 base in ref/chakraborty_simulans_header.fasta, id DNAhATAc:hAT1DP, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,221 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:BS3DM, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,222 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC2DM, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,222 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC2DM, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,222 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC2DM, not in (A,C,T,G,N): M. Changed to "N"
2024-05-08 16:36:29,222 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC2DM, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): M. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): H. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): D. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): D. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,223 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:DOC4DM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,224 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:FW2DM, not in (A,C,T,G,N): S. Changed to "N"
2024-05-08 16:36:29,224 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:FW3DM, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,225 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:G4DM, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,225 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:G5ADM, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,226 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENA, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,226 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENA, not in (A,C,T,G,N): M. Changed to "N"
2024-05-08 16:36:29,226 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENA, not in (A,C,T,G,N): M. Changed to "N"
2024-05-08 16:36:29,226 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENA, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,226 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENA, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,227 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENART, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,227 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:HELENART, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,228 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:UVIRDV, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,228 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:UVIRDV, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,228 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:UVIRDV, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,228 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:UVIRDV, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,228 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:UVIRDV, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,228 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonJockey:UVIRDV, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,230 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonR1:DMRT1A, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,230 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonR1:DMRT1A, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,230 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonR1:DMRT1A, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,231 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonR1:R12DM, not in (A,C,T,G,N): S. Changed to "N"
2024-05-08 16:36:29,231 base in ref/chakraborty_simulans_header.fasta, id NonLTRretrotransposonR1:R12DM, not in (A,C,T,G,N): S. Changed to "N"
2024-05-08 16:36:29,239 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:DMTOM1LTR, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,241 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy4LTR, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,241 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy4LTR, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,241 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy4LTR, not in (A,C,T,G,N): W. Changed to "N"
2024-05-08 16:36:29,241 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy4LTR, not in (A,C,T,G,N): K. Changed to "N"
2024-05-08 16:36:29,241 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy4LTR, not in (A,C,T,G,N): Y. Changed to "N"
2024-05-08 16:36:29,244 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy10LTR, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,244 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy11LTR, not in (A,C,T,G,N): R. Changed to "N"
2024-05-08 16:36:29,244 base in ref/chakraborty_simulans_header.fasta, id LTRGypsy:Gypsy12ALTR, not in (A,C,T,G,N): R. Changed to "N"
